src/virtual-machine.js

```diff
@@ -731,16 +734,16 @@ class VirtualMachine extends EventEmitter {
      *     a dataURI if it's a PNG or JPG, or null if it couldn't be found or decoded.
      */
     getCostume (costumeIndex) {
-        const id = this.editingTarget.getCostumes()[costumeIndex].assetId;
-        if (!id || !this.runtime || !this.runtime.storage) return null;
-        const format = this.runtime.storage.get(id).dataFormat;
+        const asset = this.editingTarget.getCostumes()[costumeIndex].asset;
+        if (!asset || !this.runtime || !this.runtime.storage) return null;
```

Comment:
Just to make sure, is there ever a case where we could have a sound/costume object in a functioning project and the `asset` property of the project is falsey?

Reply:
Maybe if there was some issue with creating assets while the project was loading? Not during normal functioning of the editor, as far as I'm aware.